.changeset/config.json

```diff
@@ -1,10 +1,15 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@3.0.4/schema.json",
-  "changelog": "@changesets/cli/changelog",
+  "changelog": [
```

Comment:
the documentation isnt super clear on this, but hoping this will fix our changesets not creating github release / tag.